Allow to set a code editor for the custom step (#419)
ToucanToco/weaverbird: CHANGELOG.md

## UNRELEASED -

### Added

> - expose `setCodeEditor` to set a code editor in the custom step

Reviewer: ```suggestion
- expose `setCodeEditor` to set a custom code editor component for  custom steps
```

> - custom step which allow user to edit manually a step

Reviewer: ```suggestion
- custom step which allows users to edit manually a step
```
